src/CLI/actioner/sonic-cli-mirror.py

```diff
@@ -0,0 +1,184 @@
+                            "Policer", "SRC Port", "Direction")
+        span_header = ("Name", "Status", "DST Port", "SRC Port", "Direction")
+
+        erspan_data = []
+        span_data = []
+        if session_name is None:
+            print("\nERSPAN Sessions")
+            print("---------------------------------------------------------------------------------------------------------")
+            print("%10s %6s %16s %16s %6s %6s %6s %6s %6s %12s %6s" %("Name", "Status", "SRC IP", "DST IP", "GRE", "DSCP", "TTL", "Queue", "Policer", "SRC Port", "Direction"))
+
+        for key, val in sessions_db_info.iteritems():
+            if session_name and key != session_name:
+                continue
+
+            if "src_ip" in val:
+                if session_name and key == session_name:
+                    print("\nERSPAN Sessions")
+                    print("---------------------------------------------------------------------------------------------------------")
+                    print("%10s %6s %16s %16s %6s %6s %6s %6s %6s %12s %6s" %("Name", "Status", "SRC IP", "DST IP", "GRE", "DSCP", "TTL", "Queue", "Policer", "SRC Port", "Direction"))
+                print("%10s %6s %16s %16s %6s %6s %6s %6s %6s %12s %6s" %(key, val.get("status", ""), val.get("src_ip", ""), val.get("dst_ip", ""), val.get("gre_type", ""), val.get("dscp", ""),
+                                         val.get("ttl", ""), val.get("queue", ""), val.get("policer", ""),
+                                                                  val.get("src_port", ""), val.get("direction", "")))
+        if session_name is None:
+            print("\nSPAN Sessions")
+            print("---------------------------------------------------------------------------------------------------------")
+            print("%10s %6s %16s %16s %6s" %("Name", "Status", "DST Port", "SRC Port", "Direction"))
+        for key, val in sessions_db_info.iteritems():
+            if session_name and key != session_name:
+                continue
+            if "dst_port" in val:
+                if session_name and key == session_name:
+                    print("\nSPAN Sessions")
+                    print("---------------------------------------------------------------------------------------------------------")
+                    print("%10s %6s %16s %16s %6s" %("Name", "Status", "DST Port", "SRC Port", "Direction"))
+                print("%10s %6s %16s %16s %6s" %(key, val.get("status", ""), val.get("dst_port", ""), val.get("src_port", ""), val.get("direction", "")))
+
+
+def session(session_name):
+    """
+    Show mirror session configuration.
+    :return:
+    """
+    show_session(session_name)
+
+def show_mirror(args):
+    """
+    Add port mirror session
+    """
+    session(args.session)
+
+def config_span(args):
+    """
+    Add port mirror session
+    """
+    config_db = ConfigDBConnector()
```

Comment:
It looks like the CLI requests directly access the Redis-DB without routing through Rest-server/Translib. Can you please confirm? if yes, why we have to divert from the existing channel?